code/game/world.dm

```diff
@@ -598,18 +598,19 @@ GLOBAL_VAR_INIT(world_topic_last, world.timeofday)
 		GLOB.log_directory += "[replacetext(time_stamp(), ":", ".")]"
 
 	GLOB.world_qdel_log = file("[GLOB.log_directory]/qdel.log")
+	GLOB.query_debug_log = file("[GLOB.log_directory]/sql.log")
 	to_file(GLOB.world_qdel_log, "\n\nStarting up round ID [game_id]. [time_stamp()]\n---------------------")
 
 #define FAILED_DB_CONNECTION_CUTOFF 5
 var/failed_db_connections = 0
 var/failed_old_db_connections = 0
 
-/hook/startup/proc/connectDB()
-	if(!setup_database_connection())
-		to_world_log("Your server failed to establish a connection with the feedback database.")
-	else
-		to_world_log("Feedback database connection established.")
-	return 1
+// /hook/startup/proc/connectDB() // Replaced by DBCore
+// 	if(!setup_database_connection())
+// 		to_world_log("Your server failed to establish a connection with the feedback database.")
+// 	else
+// 		to_world_log("Feedback database connection established.")
+// 	return 1
```

Comment:
Any reason to keep this?

Reply:
I suppose not, I presumed it useful as reference for future expansion. But yeah, that segment is rather useless.